package ru.practicum.shareit.exception.userException;

public class UserNotFoundException extends RuntimeException {
}
